### PR TITLE
Improve throughput / transparency on rule design process

### LINKED ISSUE #466: Faster turnover in rule feedback
Proposal discussed by MJ and myself. We think to get faster turnaround on rules, we should do the following:

1. When a rule is put into survey, respond in the issue when the survey is due. If the survey is extended, but that updated date in too.
2. The liaison should contact the person who proposed the rule directly to let them know when the survey is in, and ask when feedback can be resolved.

### PR DESCRIPTION
**Pull request by WilcoFiers**
This was discussed during last week's TF call.

Closes #466

### CODE REVIEW

File: wcag-ruleset-review-process.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ A facilitator of the ACT Task Force creates a survey for the rule, in which Task
...
 
-If the rule does not pass this step, the liaison has 5 work days to put the feedback in the issue for the proposal using the template in appendix 1. If changes are editorial in nature, and the rule provider can resolve those within 5 work days, the rule will be surveyed again straight away. If not, the proposal will be closed by a TF facilitator. Proposals can always be resubmitted, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
+If the rule does not pass this step, the liaison has 5 work days to put the feedback in the issue for the proposal using the template in appendix 1. When submitting the feedback, the liaison must notify the person who opened the issue and ask when the feedback can be resolved. If changes are editorial in nature, and the rule provider can resolve those within 5 work days, the rule will be surveyed again straight away. If not, the proposal will be closed by a TF facilitator. Proposals can always be resubmitted, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
```
Comment (kengdoj) [inline wcag-ruleset-review-process.md:48]: Second sentence question - the liaison (from the Task Force?) must notify the person who opened the issue (isn't this the liaison from the Task Force?) and ask when the feedback can be resolved?

Should it be - notify the person who submitted the rule for review and ask when the feedback can be resolved?

Comment (maryjom) [inline wcag-ruleset-review-process.md:48]: So is this a change in the way we're currently handling survey results? "the liaison has 5 work days to put the feedback in the issue for the proposal using the template in appendix 1."  Currently we open a new issue, reference the previous issue the survey was on and use the template to fill in the comments. This is fine, but I want to make sure.  I actually like this because the issue maintenance on our side was getting a bit hard to follow.

Perhaps the reference to the template in Appendix 1 could be a link to the appendix for easier reference.

Assuming "liaison" is the ACT TF liaison. May want to clarify that.  

I like Kathy's suggestions for edits.

File: wcag-ruleset-review-process.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ A facilitator of the ACT Task Force creates a survey for the rule, in which Task
...
 
-If the rule does not pass this step, the liaison has 5 work days to put the feedback in the issue for the proposal using the template in appendix 1. If changes are editorial in nature, and the rule provider can resolve those within 5 work days, the rule will be surveyed again straight away. If not, the proposal will be closed by a TF facilitator. Proposals can always be resubmitted, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
+If the rule does not pass this step, the liaison has 5 work days to put the feedback in a new issue for the proposal using the template in appendix 1. They must then close the issue. When submitting the feedback, the liaison must notify the person who submitted the rule and ask them when the feedback can be resolved. If changes are editorial in nature, and the rule provider can resolve those within 5 work days, the rule will be surveyed again straight away. If not, the proposal will be closed by a TF facilitator. Proposals can always be resubmitted, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
```
Comment (maryjom) [inline wcag-ruleset-review-process.md:48]:
```suggestion
If the rule does not pass this step, the liaison has 5 work days to put the feedback in a new issue for the proposal using the template in appendix 1. They must then reference the new issue in the "New rule" issue and close the "New rule" issue. When submitting the feedback, the liaison must notify the person who submitted the rule and ask them when the feedback can be resolved. If changes are editorial in nature, and the rule provider can resolve those within 5 work days, the rule will be surveyed again straight away. If not, the proposal will be closed by a TF facilitator. Proposals can always be resubmitted, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
```

Comment (maryjom) [reply wcag-ruleset-review-process.md]: @wilco We should probably make the "appendix 1" in bullet 1 above a link to appendix 1. Can you do that please?

File: wcag-ruleset-review-process.md [hunk 3]
```diff
@@ -45,7 +45,15 @@ A facilitator of the ACT Task Force creates a survey for the rule, in which Task
 
 This survey will be open for at least 5 work days, to give ACT Task Force members sufficient time to review the rule. No more than 5 rule surveys will be due in the same week.
 
-If the rule does not pass this step, the liaison has 5 work days to put the feedback in the issue for the proposal using the template in appendix 1. If changes are editorial in nature, and the rule provider can resolve those within 5 work days, the rule will be surveyed again straight away. If not, the proposal will be closed by a TF facilitator. Proposals can always be resubmitted, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
+If the rule does not pass this step, the liaison has 5 work days to:
+
+- put the feedback in a new issue using the template in appendix 1.
```
Comment (maryjom) [inline wcag-ruleset-review-process.md:50]: Would like to see the "appendix 1" be a link to the appendix.

Comment (maryjom) [reply wcag-ruleset-review-process.md]: @WilcoFiers I went ahead and made the change directly on this PR to create the link, and fixed spelling of appendix.

**Comment by WilcoFiers** [comment]
This PR is outdated, we need to rethink our process now that AG has changed its approach to reviewing.
